### ISSUE TITLE
nvda does not return voice after exiting a settings profile

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#15096]
### Steps to reproduce:
In Brazil there is a program to train typing called Digivox.
this program has its own voice.
to make the use of the program and nvda as automated as possible for students, I created a profile of nvda settings for this program called digitvox.
in the nvda settings profile i set the speech synthesizer to no speech.
Assigning the profile occurs correctly, as soon as the digitavox program is opened the nvda voice is interrupted.
The problem occurs when we close the Digivox program, the normal configuration profile must be activated and the previously configured synthesizer must start speaking.

### Actual behavior:
nvda settings perfio configured for speechless synthesizer, works correctly.
when we close the program in which this profile is configured, nvda does not speak again.
I don't know if nvda didn't return to the normal profile, or if it failed to call the speech synthesizer again.

### Expected behavior:
after the program is exited and the nvda settings profile that is using synthesizer without ffala exits execution, the normal profile using a synthesizer should be executed and the configured synthesizer should speak.
comments:
this worked correctly until the last alpha update I did, previously it was with the alpha:
alpha-28489,30e00a99
the problem does not occur if I open the program in question digitvox and close it right after.
What causes nvda to fail occurs after about 15 seconds of working on the program in question digitvox.
It could be due to the timing being relevant or something that occurs with nvda within this program.
Occurs when using sap5 or rhvoice synthesizer.
#### NVDA installed/portable/running from source:
instaled
#### NVDA version:
nvda.exe, NVDA alpha-28569,5d1b0e82

#### Windows version:
11 22621.1928
#### Name and version of other software in use when reproducing the issue:
digitavox 1
#### Does the issue still occur after restarting your computer?
yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
yes
this worked correctly until the last alpha update I did, previously it was with the alpha:
alpha-28489,30e00a99
#### If NVDA add-ons are disabled, is your problem still occurring?
yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
yes
this problem can be reproduced with any program, in any case follow the link to download the program in question.
http://intervox.nce.ufrj.br/~neno/digitavox.htm#digitavox

I couldn't get the log in a reduced way, nvda has no voice when the error occurs, but it follows the file
[digitavox.txt](https://github.com/nvaccess/nvda/files/11942935/digitavox.txt)

**Comment by thgcode** [comment]
Reproduced with Digitavox, however I think it could be related to issue #15082 .

**Comment by seanbudd** [comment]
Please reproduce this issue and provide a [log file](https://github.com/nvaccess/nvda/wiki/LogFilesAndCrashDumps) of the behaviour. Ensure your log level is set to debug in general preferences.

**Comment by seanbudd** [comment]
nevermind
